webapp/package.json

```diff
@@ -56,7 +56,8 @@
     "@nuxtjs/style-resources": "~0.1.2",
     "accounting": "~0.4.1",
     "apollo-cache-inmemory": "~1.5.1",
-    "apollo-client": "~2.6.2",
+    "apollo-client": "~2.6.1",
+    "buefy": "^0.7.7",
```

Comment:
Ohhhh, @appinteractive will not like this line of code! :laughing:

Reply:
it's what gives us our switch and checkboxes, so unless there is a plan to add them to the styleguide/ a better library, we're kinda stuck with them...

Reply:
We had bulma in the past and @appinteractive started to hate it. I guess, the best option is to maintain the styleguide now. Ie. adding features to it :slightly_frowning_face: